test/integration_tests/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# Fizzy: A fast WebAssembly interpreter
+# Copyright 2020 The Fizzy Authors.
+# SPDX-License-Identifier: Apache-2.0
+
+add_test(
```

Comment:
Hmm, couldn't we run spectest smoketests the same way?

Reply:
Sure. I have not connected the dots..